### ISSUE TITLE
Unable to fetch Index Quotes

### ISSUE THREAD

**Issue by n1ghthawk** [samco-sdk/Python-SDK#7]
The API doesn't seem to have integrated function to fetch index quotes (Eg: NIFTY, BANKNIFTY, FINNIFTY). get_quote() doesn't fetch Index quotes. 

Please look into it.

**Comment by KPMR-PYTHON** [comment]
Yes.. please provide details of getting / fetching  index quotes (Eg: NIFTY, BANKNIFTY, FINNIFTY)..
